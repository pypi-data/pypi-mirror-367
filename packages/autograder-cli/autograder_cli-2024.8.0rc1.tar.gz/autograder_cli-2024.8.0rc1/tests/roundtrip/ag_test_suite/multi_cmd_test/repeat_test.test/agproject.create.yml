project:
  name: Test Project d062eb1ea54346a58e3617f8845d0220
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings:
  test_suites:
  - name: Suite
    test_cases:
    - name: Test $test_name
      commands:
      - name: Compile
        cmd: ./compile.sh $test_name
      - name: Run
        cmd: ./run.sh $test_name
      - name: Check
        cmd: ./check.sh $test_name
      repeat:
      - $test_name: spam
      - $test_name: egg
      - $test_name: sausage
