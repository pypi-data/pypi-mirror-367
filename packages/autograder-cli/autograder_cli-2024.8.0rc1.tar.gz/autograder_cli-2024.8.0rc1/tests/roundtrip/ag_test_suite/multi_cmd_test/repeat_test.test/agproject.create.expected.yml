project:
  name: Test Project d062eb1ea54346a58e3617f8845d0220
  timezone: America/Chicago
  course:
    name: Test Course
    semester: Summer
    year: 2014
  settings: {}
  test_suites:
  - name: Suite
    test_cases:
    - name: Test spam
      commands:
      - name: Compile
        cmd: ./compile.sh spam
      - name: Run
        cmd: ./run.sh spam
      - name: Check
        cmd: ./check.sh spam
    - name: Test egg
      commands:
      - name: Compile
        cmd: ./compile.sh egg
      - name: Run
        cmd: ./run.sh egg
      - name: Check
        cmd: ./check.sh egg
    - name: Test sausage
      commands:
      - name: Compile
        cmd: ./compile.sh sausage
      - name: Run
        cmd: ./run.sh sausage
      - name: Check
        cmd: ./check.sh sausage
